dcos_launch/onprem.py

```diff
@@ -34,9 +35,11 @@ def get_last_state(self):
     def get_bare_cluster_launcher(self):
         if self.config['platform'] == 'aws':
             return dcos_launch.aws.BareClusterLauncher(self.config)
+        elif self.config['platform'] == 'gce':
+            return dcos_launch.gce.BareClusterLauncher(self.config)
         else:
-            raise dcos_launch.util.LauncherError('Platform currently not supported for onprem: {}'.format(
-                self.config['platform']))
+            raise dcos_launch.util.LauncherError('PlatformNotSupported', """Platform currently not supported for onprem:
+                                                 {}""".format(self.config['platform']))
```

Comment:
You're right, can't do hanging-indent w/ something on the first line. My example should have been:

```
raise dcos_launch.util.LauncherError(
    'PlatformNotSupported',
    'Platform currently not supported for onprem: {}'.format(self.config['platform']))
```
Your choice. Doesn't really matter to me.